packages/column/Column.scss

```diff
@@ -0,0 +1,111 @@
+@mixin one-column {
+  grid-template-columns: 1fr;
+}
+@mixin two-column {
+  grid-template-columns: repeat(2, 1fr);
+}
+@mixin three-column {
+  grid-template-columns: repeat(3, 1fr);
+}
+@mixin four-column {
+  grid-template-columns: repeat(4, 1fr);
+}
+
+.columnWrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 24px;
+  border: 1px solid #eaeaec;
+  max-width: 1352px;
+  margin: 0 auto;
+}
+
+@media (min-width: 664px) {
+  .small-1 {
+    @include one-column;
+  }
+
+  .small-2 {
+    @include two-column;
+  }
+
+  .small-3 {
+    @include three-column;
+  }
+
+  .small-4 {
+    @include four-column;
+  }
+}
+
+@media (min-width: 1008px) {
+  .medium-1 {
+    @include one-column;
+  }
+
+  .medium-2 {
+    @include two-column;
+  }
+
+  .medium-3 {
+    @include three-column;
+  }
+
+  .medium-4 {
+    @include four-column;
+  }
+}
+
+@media (min-width: 1080px) {
+  .large-1 {
+    @include one-column;
+  }
+
+  .large-2 {
+    @include two-column;
+  }
+
+  .large-3 {
+    @include three-column;
+  }
+
+  .large-4 {
+    @include four-column;
+  }
+}
+
+@media (min-width: 1157px) {
+  .xlarge-1 {
+    @include one-column;
+  }
+
+  .xlarge-2 {
+    @include two-column;
+  }
+
+  .xlarge-3 {
+    @include three-column;
+  }
+
+  .xlarge-4 {
+    @include four-column;
+  }
+}
+
+@media (min-width: 1495px) {
```

Comment:
I don't think these classes will be used, unless you also add a `xxlarge` prop to the component.